cmd/repl/repl.go

```diff
@@ -11,13 +11,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package repl
 
 import (
 	"bufio"
 	"bytes"
 	"image"
-	_ "image/png"
+	_ "image/png" // comment(yancey1989): why import this blank import
```

Comment:
Using the side effect to register functions that processes `png` is a typical usage of the `image` module.

Reply:
I moved the common code to `pkg/step`, and add some tests.